src/main/java/com/thatredhead/redbot/command/impl/EvalCommands.java

```diff
@@ -0,0 +1,48 @@
+package com.thatredhead.redbot.command.impl;
+
+import com.thatredhead.redbot.command.Command;
+import com.thatredhead.redbot.command.CommandGroup;
+import com.thatredhead.redbot.helpers4d4j.MessageParser;
+import com.thatredhead.redbot.helpers4d4j.Utilities4D4J;
+import com.thatredhead.redbot.permission.PermissionContext;
+import org.jetbrains.kotlin.script.jsr223.KotlinJsr223JvmDaemonLocalEvalScriptEngineFactory;
+import sx.blah.discord.util.EmbedBuilder;
+
+import javax.script.ScriptException;
+import java.util.ArrayList;
+
+public class EvalCommands extends CommandGroup {
+    public EvalCommands() {
+        super("eval", "evaluates a kotlin script", "developer", new ArrayList<>());
+    }
+
+
+    public static class Eval extends Command {
+        private KotlinJsr223JvmDaemonLocalEvalScriptEngineFactory factory = new KotlinJsr223JvmDaemonLocalEvalScriptEngineFactory();
+
+        public Eval(String keyword, String description, PermissionContext defaultPerm) {
+            super(keyword, description, PermissionContext.BOT_OWNER);
```

Comment:
The constructor should be no args default. This is where you specify the command's keyword and description and stuff. Essentially much of what you had in the EvalCommands constructor.